realm/realm-library/src/androidTest/java/io/realm/internal/test/ColumnTypeData.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Realm Inc.
```

Comment:
Has files of internal test been created in 2014? When I used `git log`, there were no history before 2015. If they had been imported from other directory, I should modify them all as 2014.